### ISSUE TITLE
how to calculate "normalized latency"

### ISSUE THREAD

**Issue by Mr-lonely0** [raymin0223/fast_robust_early_exit#5]
Hi!

Thanks for your wonderful work. I was just wondering how to calculate the "normalized latency" which is not stated specifically in the paper.

All the best

**Comment by raymin0223** [comment]
Hi @Mr-lonely0,

Thanks for your interests, and sorry for the confusion.
We calculated that metric by dividing the target latency with the latency of full model (using all layers of model, which is fine-tuned with weighted cross-entropy loss; the rightmost green point in Figure 5.)

**Comment by Mr-lonely0** [comment]
Thanks for your detailed explanation!!!

I have a follow-up question regarding model latency. How can I determine the latency of a model, is it measured by the inference time required to produce an output of a specific length or the FLOPs the model performs after layer skipping, or is it perhaps neither of these and something else entirely?

I would be grateful for your response :)

**Comment by raymin0223** [comment]
As we measured "latency", we measure the total inference time required to finish all requests (all test or validation samples in our case). Thus, we don't care if different models outputs different length of tokens. 
If you want to consider the output length, it would be good to measure throughput (token/s) instead of latency.

**Comment by Mr-lonely0** [comment]
Thanks again!

In this way, layer skipping may generate longer outputs(token repetition) and thus more skipped layers may lead to greater latency. So how can we ensure that reducing latency correlates with more efficient inference?

**Comment by raymin0223** [comment]
Since we are looking at the Pareto frontier of latency to ROUGE-L scores, I believe we can just ensure that models output reasonable predictions with similar sequence lengths, when they have similar performance!
